CHANGES

```diff
@@ -1,7 +1,7 @@
 DD MMM YYYY - 2.9.2 - To be released
 ------------------------------------
 
- * {dis|en}able-dechunking: Option to disable logging of
```

Comment:
This is a very simple an compact patch. Please avoid to make a commit to fix your own commit. Use ```git commit --amend``` instead.